cmd/lotus-chainwatch/processor/reward.go

```diff
@@ -280,6 +280,9 @@ func (p *Processor) storeRewardSmoothingEstimates(rewards []rewardActorInfo) err
 	}
 
 	for _, rewardState := range rewards {
+		if rewardState.epochSmoothingEstimate == nil {
+			continue
```

Comment:
If this _is_ the issue I'm seeing in #3183, we should probably _fail_ here instead of just continuing.

Reply:
We want to be tolerant to imperfect and/or missing data and prefer to accept the loss and move on rather than kill the entire process. Although, if we'd like to be better informed of these cases, capturing the failure for later action seems reasonable.